package.java.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [[ "$VERSION" == *java ]]
+then
+    mvn install
+else
+    cp ./grpc/cubic.proto ./grpc/android-cubic/src/main/proto/cubic.proto
+    cd ./grpc/android-cubic && chmod +x ./gradlew && ./gradlew clean install -xtest
+fi
```

Comment:
IS the $VERSION pulling the git tag?  I wasn't clear where this was coming from.

It might be nice to add comments like this:
```suggestion
if [[ "$VERSION" == *java ]]
then
    # Build the plain Java version
    mvn install
else
  # Build the Android gradle version
    cp ./grpc/cubic.proto ./grpc/android-cubic/src/main/proto/cubic.proto
    cd ./grpc/android-cubic && chmod +x ./gradlew && ./gradlew clean install -xtest
fi
```